examples/image_segmentation/ios/ImageSegmenter/Configurations/DefaultConstants.swift

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import UIKit
+
+// MARK: Define default constants
+struct DefaultConstants {
+
+  static let lineWidth: CGFloat = 2
```

Comment:
Except for the model path, remove the other constants since they seem to be unused